niceman/distributions/tests/test_conda.py

```diff
@@ -20,108 +20,212 @@
 
 import yaml
 import attr
+
 from niceman.formats.niceman import NicemanProvenance
 from niceman.tests.utils import create_pymodule
 from niceman.tests.utils import skip_if_no_network, assert_is_subset_recur
 
 import json
 
-from niceman.distributions.conda import CondaTracer, \
-    get_conda_platform_from_python
-
-
-@pytest.fixture(scope="session")
-@skip_if_no_network
-def get_conda_test_dir():
-    dirs = AppDirs('niceman')
-    test_dir = os.path.join(dirs.user_cache_dir, 'conda_test')
-    if os.path.exists(test_dir):
-        return test_dir
-    # Miniconda isn't installed, so install it
-    if sys.platform.startswith('darwin'):
-        miniconda_sh = "Miniconda2-latest-MacOSX-x86_64.sh"
-    elif sys.platform.startswith('linux'):
-        miniconda_sh = "Miniconda2-latest-Linux-x86_64.sh"
-    else:
-        raise ValueError("Conda test not supported with platform %s " %
-                         sys.platform)
-
-    pymod_dir = os.path.join(test_dir, "minimal_pymodule")
-    create_pymodule(pymod_dir)
```

Comment:
[mostly thinking out loud]

Dropping this means that we're not testing that editable packages are detected when we trace conda environments. I'm guessing this is necessary because we're now doing an instantiate then trace test, and we can't instantiate editable packages (at least with conda instantiation alone).

Reply:
Ah...  but I can squeeze in the creation of the minimal_pymodule after the instantiate (I do something similar in order to test the detection of an empty conda environment).  I'll look into that.

Reply:
I was able to add the minimal_pymodule test back in.